### DISCUSSION TITLE
Container Elements still broken?

### DISCUSSION THREAD

**Discussion by jake-macadamian** [floating-ui/floating-ui#2970] [General]
Hi, I saw a bunch of issues around containers getting closed recently so I didn't want to make a new bug report if it's coming down the pipe.  I'm using @floating-ui/dom@1.6.7 and it's calculating the wrong position.
```
<div> -- container-type: inline-size
   <div popovertarget></div> -- Target Element
   <div popover></div> -- Popover Element
</div>
```
It calculates the position as if the container were at the top left corner of the screen.  When I disable the container-type it calculates the position correctly.
Should I wait for version 1.6.8?  or should I report this as a bug?

**Comment by atomiks** [comment]
It's okay to make a bug report with a reproduction even if it's a duplicate especially if it's not in a release. I can only think of https://github.com/floating-ui/floating-ui/issues/2955 being related

**Reply by jake-macadamian** [reply]
Converted it to an issue, thanks 🙂  https://github.com/floating-ui/floating-ui/issues/2971
